components/resources/brave_components_strings.grd

```diff
@@ -390,8 +390,11 @@
       <message name="IDS_BRAVE_NEWS_FOLLOW_BUTTON_NOT_FOLLOWING" desc="Label for the follow button when the publisher is not currently followed">
         Follow
       </message>
-      <message name="IDS_BRAVE_NEWS_SEARCH_RESULTS_LOADING" desc="Label displayed while search results are loading">
-        Loading...
+      <message name="IDS_BRAVE_NEWS_DIRECT_SEARCH_BUTTON" desc="Label for the button which will search a Url for a feed">
+        Get feeds from <ph name="URL">$1<ex>https://example.com</ex></ph>
```

Comment:
Do we want to display the url in the button? It means the size of the button is going to be pretty variable

Reply:
I think for now, yes. And I'm only using a button because we don't have a tertiary button yet

Reply:
:+1: